src/compute-client/src/sinks.proto

```diff
@@ -0,0 +1,42 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+import "repr/src/antichain.proto";
+import "repr/src/global_id.proto";
+import "repr/src/relation_and_scalar.proto";
+import "storage/src/controller.proto";
+
+package mz_compute_client.sinks;
+
+message ProtoComputeSinkDesc {
```

Comment:
Can we remove these from the storage proto, or are they still referenced somewhere?

Reply:
oof -- good catch, thanks!